diff --git a/node_modules/@kingstinct/react-native-activity-kit/ios/Utils.swift b/node_modules/@kingstinct/react-native-activity-kit/ios/Utils.swift
index 52f466c..3feb6b0 100644
--- a/node_modules/@kingstinct/react-native-activity-kit/ios/Utils.swift
+++ b/node_modules/@kingstinct/react-native-activity-kit/ios/Utils.swift
@@ -48,8 +48,8 @@ func getAnyMapValue(_ anyMap: AnyMap, key: String) -> Any? {
     if anyMap.isString(key: key) {
         return anyMap.getString(key: key)
     }
-    if anyMap.isBigInt(key: key) {
-        return anyMap.getBigInt(key: key)
+    if anyMap.isInt64(key: key) {
+        return anyMap.getInt64(key: key)
     }
     if anyMap.isNull(key: key) {
         return nil
@@ -82,7 +82,7 @@ func serializeAnyMap(_ metadata: [String: Any]?) -> AnyMap {
             }
 
             if let integer = item.value as? Int64 {
-                serialized.setBigInt(key: item.key, value: integer)
+                serialized.setInt64(key: item.key, value: integer)
             }
 
             if let dict = item.value as? [String: AnyValue] {
